tests/test.py

```diff
@@ -0,0 +1,37 @@
+import unittest
+import json as js
+from urllib.request import urlopen
+from bs4 import BeautifulSoup
+from engine.songslover import Songslover as Engine
+
+
+class Test(unittest.TestCase,Engine):
```

Comment:
your test cases aren't exactly testing anything

Reply:
yeah . I know.. i made it generic. i will update it to test songslover shortly

Reply:
the engines should have their tests sha

Reply:
Yeah i wanted to ask , pytest or unittest?

Reply:
??

Reply:
Anyone is fine bro

Reply:
got it